docs: make |lspconfig-servers| jump to the markdown file (#1491)
neovim/nvim-lspconfig: README.md

    npm i -g pyright
    ```

> 2. Add the language server setup to your init.vim. The server name must match those found in the table of contents in [server_configurations.md](doc/server_configurations.md) `:help lspconfig-server-configurations`

Reviewer: "accessible via"

> 2. Add the language server setup to your init.vim. The server name must match those found in the table of contents in [server_configurations.md](doc/server_configurations.md) accessible via `:help lspconfig-server-configurations`

Reviewer 2: ```suggestion
2. Add the language server setup to your init.vim. The server name must match those found in the table of contents in [server_configurations.md](doc/server_configurations.md). This list is also accessible via `:help lspconfig-server-configurations`.
```

EOF
```

> For a full list of servers, see [server_configurations.md](doc/server_configurations.md) `:help lspconfig-server-configurations`. This document contains installation instructions and additional, optional customization suggestions for each language server. For some servers that are not on your system path (jdtls, elixirls) you will be required to manually add `cmd` as an entry in the table passed to setup. Most language servers can be installed in less than a minute.

Reviewer: "accessible via"

> For a full list of servers, see [server_configurations.md](doc/server_configurations.md) accessible via `:help lspconfig-server-configurations`. This document contains installation instructions and additional, optional customization suggestions for each language server. For some servers that are not on your system path (jdtls, elixirls) you will be required to manually add `cmd` as an entry in the table passed to setup. Most language servers can be installed in less than a minute.

Reviewer 2: no need to repeat here? Or simply "see [...](...) or `:help ...`.

> For a full list of servers, see [server_configurations.md](doc/server_configurations.md) accessible via `:help lspconfig-server-configurations`. This document contains installation instructions and additional, optional customization suggestions for each language server. For some servers that are not on your system path (jdtls, elixirls) you will be required to manually add `cmd` as an entry in the table passed to setup. Most language servers can be installed in less than a minute.

Reviewer 2: ```suggestion
For a full list of servers, see [server_configurations.md](doc/server_configurations.md) or `:help lspconfig-server-configurations`. This document contains installation instructions and additional, optional, customization suggestions for each language server. For some servers that are not on your system path (e.g., `jdtls`, `elixirls`), you will be required to manually add `cmd` as an entry in the table passed to `setup`. Most language servers can be installed in less than a minute.
```
